tools/rna_tools/targetfinder/TargetFinder_threads.xml

```diff
@@ -0,0 +1,57 @@
+<tool id="targetfinder_threads" name="targetfinder_threads" version="0.1.0">
```

Comment:
Than lets go with this tool here and abandon the other tool. From a user point of view this makes no difference and everything should be a file in Galaxy to reuse this tool as one piece in a workflow.